.cspell/en-words.txt

```diff
@@ -35,6 +35,7 @@ daemonset
 datadog
 discoverability
 dotnet
+Dubbo
```

Comment:
Hi @lmolkova! These spelling exceptions should actually be added to the front matter of your blog post. We keep the .cspell file for widely used words only. I'll add a comment in the blog file to show you where you can add them there, and then you can revert the changes to this file.